__tests__/app.test.js

```diff
@@ -1,10 +1,43 @@
-const {} = require(`${__dirname}/../db/app.js`)
+const request = require('supertest');
+const db = require(`${__dirname}/../db/connection.js`);
+const seed = require(`${__dirname}/../db/seeds/seed.js`);
+const data = require(`${__dirname}/../db/data/test-data/index.js`);
+const app = require(`${__dirname}/../db/app.js`);
+
+
+beforeEach(() => {
+    return seed(data);
+});
+
+afterAll(() => {
+    return db.end();
+});
 
 
 describe('app.js test suite', () => {
+
     describe('GET /api/topics', () => {
-        test('should ', () => {
-            
+        test('responds with status code 200', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
         });
+
+        test('responds with an array of topic objects with slug and description properties', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body}) => {
```

Comment:
There should be an assertion that the length of the array is what you should expect it to be.